invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/components/SendMailModal/SendMailModal.js

```diff
@@ -0,0 +1,84 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import { Header, Modal, Button, Icon } from 'semantic-ui-react';
+import { BackOfficeRoutes } from '../../../../routes/urls';
+
+export default class SendMailModal extends Component {
+  state = { isModalOpen: false };
+
+  toggle = () => this.setState({ isModalOpen: !this.state.isModalOpen });
+
+  sendMail = async () => {
+    const { loan } = this.props;
+    this.toggle();
+    await this.props.sendOverdueLoansMailReminder({
```

Comment:
is there any reason that we have `await` here since there is already one in the action?

Reply:
Nope its a left over :/ !